src/components/header/_macro-options.md

```diff
@@ -61,6 +62,18 @@
 | itemsList            | array`<Item>`          | true     | Settings for an array of [list items](#item) for each navigation link                                                                 |
 | toggleServicesButton | object`<ToggleButton>` | true     | Settings for the [mobile service links toggle button](#togglebutton)                                                                  |
 
+## NavLinks
+
+| Name            | Type                   | Required | Description                                                                                                                           |
+| --------------- | ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| id              | string                 | true     | The HTML `id` of the `<nav>` element. Used for the `aria-controls` attribute for the menu toggle button displayed on small viewports. |
+| classes         | string                 | false    | Classes to add to the `<nav>` element                                                                                                 |
+| ariaLabel       | string                 | false    | The `aria-label` attribute added to the `<nav>` element. Defaults to “Menu links navigation”.                                         |
+| ariaListLabel   | string                 | false    | The `aria-label` attribute added to the `<ul>` element. Defaults to “Menu links”.                                                     |
+| keyLinksList    | array`<Item>`          | true     | Settings for an array of [key list items](#keyitem) for each navigation link                                                          |
+| itemsList       | array`<Item>`          | true     | Settings for an array of [list items](#item) for each navigation link                                                                 |
```

Comment:
Shouldnt this be linksList? But I actually think this isn't descriptive enough. Maybe it should be
menuLinks > menuGroup > Items I think thats a bit more descriptive to the user what that is referring to

Reply:
Looking at it I think we can rename "menuGroup" to "menuCol" and then the "subItems" could be "groupItems". Then we would have the menu columns named as such and the same for the groups within them. I think this makes more sense to me because it relates more to what is on screen